app/src/cli/commands/clone.ts

```diff
@@ -0,0 +1,47 @@
+import * as QueryString from 'querystring'
+import { URL } from 'url'
+
+import { CommandError } from '../util'
+import { openDesktop } from '../open-desktop'
+import { ICommandModule, mriArgv } from '../load-commands'
+
+interface ICloneArgs extends mriArgv {
+  readonly branch?: string
+}
+
+const command: ICommandModule = {
+  command: 'clone <url|slug>',
+  description: 'Clone a repository',
+  args: [
+    {
+      name: 'url|slug',
+      required: true,
+      description: 'The URL to clone, or the GitHub repo slug to clone',
+      type: 'string',
+    },
+  ],
+  options: {
+    branch: {
+      type: 'string',
+      aliases: ['b'],
+      description: 'The branch to switch to after cloning',
```

Comment:
Let's use Git terminology here and tighten it up a bit:

> The branch to checkout after cloning

Reply:
✅